apps/address-book/package.json

```diff
@@ -16,8 +16,8 @@
     "precommit": "lint-staged",
     "prepublishOnly": "truffle compile --all",
     "publish": "aragon apm publish patch && sleep 3",
-    "publish:cd": "aragon apm publish major --files dist/ --environment continuous-deployment --apm.ipfs.rpc https://ipfs.autark.xyz:5001 --ipfs-check false",
-    "publish:http": "npm run build:script && aragon apm publish major --http localhost:1111 --http-served-from ./dist",
+    "publish:cd": "yes | aragon apm publish major --files dist/ --environment continuous-deployment --apm.ipfs.rpc https://ipfs.autark.xyz:5001 --ipfs-check false --skip-confirmation true",
```

Comment:
I guess if we add the skip confirmation option we shouldn't need the `yes` solution, but it shouldn't harm to have both for now, so why not?

Reply:
We do need both, unfortunately. With only `--skip-confirmation true`, it still gets stuck at this step:

    Couldn't reuse artifact due to mismatches, regenerate now? [y]es/[a]bort

I consider it a bug in aragonCLI that `--skip-confirmation` doesn't apply here, and I'd like to see one of us submit a patch soon.